delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/disk/DiskStorageSaveFile.scala

```diff
@@ -70,13 +70,23 @@ object DiskStorageSaveFile {
       disk: DiskStorageValue,
       uuid: UUID,
       filename: String
+  ): IO[(Path, Path)] =
+    for {
+      (resolved, relative) <- computeLocation(project, disk, uuid, filename)
+      dir                   = resolved.getParent
+      _                    <- IO.blocking(Files.createDirectories(dir)).adaptError(couldNotCreateDirectory(dir, _))
```

Comment:
Maybe it's worth Files.createDirectories be blocking rather than needing the caller to block

Reply:
That's a Java method (ideally it should use fs2 but I didn't want to change this everywhere here)